models/breast_density_classification/large_files.yml

```diff
@@ -0,0 +1,5 @@
+large_files:
+  - path: "models/model.pt"
```

Comment:
Could you please help to double check the weights? When I run the inference script, all outputs are [1, 0, 0, 0]